resources/js/src/app/components/basket/BasketTotals.js

```diff
@@ -16,6 +16,13 @@ Vue.component("basket-totals", {
         showNetPrices: state => state.basket.showNetPrices
     }),
 
+    methods: {
+        calculation: function(value, percent)
```

Comment:
Schreib das mal so calculation(value, percent). Und nenn die Methode mal so das man besser versteht was in dieser passiert oder mach vllt ein Kommentar drüber.

Reply:
ich würde hier sogar eher eine computedProperty verwenden. Macht das Template etwas lesbarer.